test/plugins/scenarios/fasthttp/plugin.yml

```diff
@@ -0,0 +1,35 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+entry-service: http://${HTTP_HOST}:${HTTP_PORT}/consumer
+health-checker: http://${HTTP_HOST}:${HTTP_PORT}/health
+start-script: ./bin/startup.sh
+framework: github.com/valyala/fasthttp
+export-port: 8080
+support-version:
+  - go: 1.18
+    framework:
+      - v1.40.0
+      - v1.41.0
+      - v1.42.0
+      - v1.43.0
+      - v1.44.0
+      - v1.45.0
+      - v1.46.0
+      - v1.47.0
+      - v1.48.0
+      - v1.49.0
+      - v1.50.0
```

Comment:
Why don't test 1.10, 1.20, 1.30, 1.40, 1.50? I think that would be more reliable.

Reply:
Okay, I have modified the configuration